wizard/WizardDaemonSettings.qml

```diff
@@ -92,6 +92,34 @@ ColumnLayout {
                     blockchainFolder.focus = true;
                 }
             }
+            onTextUpdated: {
+                var dbExists = daemonManager ? daemonManager.checkLmdbExists(blockchainFolder.text) : false
+                existingDbWarning.visible = dbExists
+                if (dbExists)
+                    pruneBlockchainCheckBox.checked = false;
+            }
```

Comment:
```suggestion
```

Seems redundant, works without this code here too.

existingDbWarning.visible should get updated once blockchainFolder.text changes, and pruneBlockchainCheckBox.checked also gets updated once existingDbWarning.visible changes.